GearSwapper/src/main/java/net/runelite/client/plugins/gearswapper/GearSwapper.java

```diff
@@ -322,6 +322,31 @@ private void executeSwap(List<WidgetItem> equip, List<Widget> remove)
 		return equipped;
 	}
 
+	private void spec()
+	{
+		if (client.getWidget(WidgetInfo.MINIMAP_SPEC_SPRITE).getSpriteId() == 1607 || client.getWidget(WidgetInfo.MINIMAP_SPEC_SPRITE).getSpriteId() == 1608)
+		{
+			handleSwitch(client.getWidget(WidgetInfo.MINIMAP_SPEC_ORB).getBounds());
```

Comment:
Widgets are nullable, so its best to nullcheck, even if its highly unlikely to null.

Reply:
Should be fixed now mb